src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MathFunction.cs

```diff
@@ -66,17 +66,21 @@ public override bool CheckInvocation(TexlNode[] args, DType[] argTypes, IErrorCo
             var arg = args[0];
             var argType = argTypes[0];
             fValid &= CheckNumericColumnType(argType, arg, errors, ref nodeToCoercedTypeMap);
+            DType dType;
 
             if (nodeToCoercedTypeMap?.Any() ?? false)
             {
                 // Now set the coerced type to a table with numeric column type with the same name as in the argument.
-                returnType = nodeToCoercedTypeMap[arg];
+                dType = nodeToCoercedTypeMap[arg];
             }
             else
             {
-                returnType = argType;
+                dType = argType;
             }
 
+            var typedName = new TypedName(dType, new DName("Value"));
+            returnType = DType.CreateTable(typedName);
```

Comment:
This is almost certainly a semantic breaking change. Should it be using Flags?

Reply:
Yeah, it's going to be in draft mode till the PA-Client doc converter changes are complete.

Reply:
Yes, we'll definitely need changes in PAClient, including a converter there. Flag is a good idea, to give us time to check this change in PFx and have time to do the change in PAClient